taichi/llvm/llvm_context.cpp

```diff
@@ -368,6 +368,9 @@ std::unique_ptr<llvm::Module> TaichiLLVMContext::clone_module(
     patch_intrinsic("cuda_shfl_up_sync_i32", Intrinsic::nvvm_shfl_sync_up_i32);
     patch_intrinsic("cuda_shfl_up_sync_f32", Intrinsic::nvvm_shfl_sync_up_f32);
 
+    patch_intrinsic("cuda_shfl_xor_sync_i32",
+                    Intrinsic::nvvm_shfl_sync_xor_i32);
```

Comment:
```suggestion
                    Intrinsic:: nvvm_shfl_sync_bfly_i32);
```
There doesn't seem to be `Intrinsic::nvvm_shfl_sync_xor_i32` :-)

Reply:
Thanks a lot, I have no access to a cuda environment, so it's not easy to correct errors, I'm currently looking for some cloud platforms for cuda environment.